docs/topics/mta-web-downloading-an-analysis-report.adoc

```diff
@@ -6,6 +6,10 @@
 [id="mta-web-downloading-an-analysis-report_{context}"]
 = Downloading an analysis report
 
+An {ProductShortName} analysis report contains a number of sections, including a listing of the technologies used by the application, the dependencies of the application, and the lines of code that must be changed to successfully migrate or modernize the application.
+
+For more information about the contents of an {ProductShortName} analysis report, see link:{ProductDocUserGuideURL}#review-reports_cli-guide[Reviewing the reports].
```

Comment:
To be discussed whether this reference must be removed or not

Reply:
Thanks @mpershina 
I just checked the static analysis report that can be downloaded from MTA UI and this reference looks relevant

Reply:
Oki :-) Let's leave it as it is then.